src/Command/InvalidatePathCommand.php

```diff
@@ -39,7 +36,7 @@ class InvalidatePathCommand extends BaseInvalidateCommand
      */
     public function __construct(CacheManager $cacheManager = null, $commandName = 'fos:httpcache:invalidate:path')
     {
-        $this->commandName = $commandName;
+        static::$defaultName = $commandName;
```

Comment:
IMO it does not make sense to use the static property for this. The static property is only used before the command is instantiated. Setting it in the constructor doesn't do anything.

Reply:
I would also deprecate passing the commandName argument in the constructor.

Reply:
yes, setting the name in the constructor was a weird idea of flexibility. i wanted to deprecate that, willdo.

about the static property: i understood that this is the sf 4 way to do the command name, with lazy services. i explicitly define the property so that this works with older symfony versions that do not have the property.

Reply:
setting the default on the static property now, and updated the doc to say using this parameter is deprecated